android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -304,12 +304,6 @@ public TiUIWebView(TiViewProxy proxy)
 			settings.setDatabaseEnabled(true);
 		}
 
-		File cacheDir = TiApplication.getInstance().getCacheDir();
-		if (cacheDir != null) {
-			settings.setAppCacheEnabled(true);
-			settings.setAppCachePath(cacheDir.getAbsolutePath());
-		}
```

Comment:
Can you add some note why this needs to be removed?

Reply:
because Google removed the methods :smile: 
![Screenshot_20220818_145657](https://user-images.githubusercontent.com/4334997/185400281-a90d651b-2f77-4d6e-8c49-e718c7a57c7b.png)

Reply:
I'm just wondering what this means for the usage of web views. Maybe it's just something they do internally now? But if it requires permissions for examples, that should be adopted.

Reply:
Found some more infos in Studio:
> The Application Cache API is deprecated and this method will become a no-op on all Android versions once support is removed in Chromium. Consider using Service Workers instead. See https://web.dev/appcache-removal/ for more information.

and there they say 
> In light of that, Chrome WebView will support AppCache without an origin trial until the final removal takes place, expected in Chrome 90.